### ISSUE TITLE
wrong aws region for the aws-client and caused failure of creating credential

### ISSUE THREAD

**Issue by moyun** [hortonworks/cloudbreak#5167]
- Cloudbreak version  : 2.9.1
- Host platform: amazon linux 2
 
I am in Tokyo region and made the following Profile before installed the CBD:
```
export UAA_DEFAULT_SECRET=[my-secret]
export UAA_DEFAULT_USER_PW=[my password]
export UAA_DEFAULT_USER_EMAIL=[my email]
export PUBLIC_IP=[my ip]
export CB_AWS_VPC=[my vpc id]
```

after executed the command: `cbd generate`, it cloud generate the correct `aws region` and other info in `Profile`
such as:
```
export CB_INSTANCE_UUID=[...]
export CB_INSTANCE_NODE_ID=[...]
export CB_INSTANCE_PROVIDER=aws
export CB_INSTANCE_REGION=ap-northeast-1b
```
However, when I tried to create the credential, it always throws an error message, said: Could not verify keys ... cannot execute the HTTP request against "eu-west-1".
I believe the cbd used the default aws region defined in AwsDefaultZoneProvider.java.
The AwsCredentialConnector did not load the correct aws region information.

**Comment by yuliu-sn** [comment]
![Screen Shot 2019-06-03 at 16 25 20](https://user-images.githubusercontent.com/42429725/58783608-47018900-861c-11e9-9d90-44df5bcf8930.png)
The error message from CB WEB UI.

**Comment by yuliu-sn** [comment]
is it correct that the http request was sent to `ec2.eu-west-1` ? 
It seems the verify function did not get the correct region form the environment variables.

**Comment by lacikaaa** [comment]
if I'm right we never used it for such purpose. Also it has been removed totally in 2.9, only left it in the deployer, but has no effect anymore.

**Comment by yuliu-sn** [comment]
@lacikaaa hi do you mean that users cannot create (actually import) AWS credential from CB GUI?

**Comment by lacikaaa** [comment]
no, I didn't mean that. `CB_INSTANCE_REGION` is not supported anymore.
